cli/public_key/cli.py

```diff
@@ -44,14 +82,14 @@ def get_public_keys(address, node_url):
         print_errors(errors)
         sys.exit(FAILED_EXIT_FROM_COMMAND_CODE)
 
-    address = arguments.get('address')
+    account_address = arguments.get('address')
     node_url = arguments.get('node_url')
 
     remme = Remme(network_config={
         'node_address': str(node_url) + ':8080',
     })
 
-    result, errors = PublicKey(service=remme).get_list(address=address)
+    result, errors = PublicKey(service=remme).get_list(address=account_address)
```

Comment:
Please, add this type of changes to the technical debt instead of fixing it in not related pull requests.